podcasts/PlaybackManager.swift

```diff
@@ -401,18 +401,44 @@ class PlaybackManager: ServerPlaybackDelegate {
         seekingTo != PlaybackManager.notSeeking
     }
 
-    func seekTo(time: TimeInterval, startPlaybackAfterSeek: Bool = false) {
-        seekTo(time: time, syncChanges: SyncManager.isUserLoggedIn(), startPlaybackAfterSeek: startPlaybackAfterSeek)
+    func seekTo(time: TimeInterval, startPlaybackAfterSeek: Bool = false, seekHint: SeekHint? = nil) {
+        seekTo(time: time, syncChanges: SyncManager.isUserLoggedIn(), startPlaybackAfterSeek: startPlaybackAfterSeek, seekHint: seekHint)
     }
 
     func seekToFromSync(time: TimeInterval, syncChanges: Bool, startPlaybackAfterSeek: Bool) {
         analyticsPlaybackHelper.currentSource = .sync
         seekTo(time: time, syncChanges: syncChanges, startPlaybackAfterSeek: startPlaybackAfterSeek)
     }
 
-    func seekTo(time: TimeInterval, syncChanges: Bool, startPlaybackAfterSeek: Bool = false) {
+    private var previousSeekTime: TimeInterval?
+
+    enum SeekHint {
+        case back
+        case forward
+    }
+
+    private let debouncer = Debounce(delay: 1.second)
+
+    func seekTo(time: TimeInterval, syncChanges: Bool, startPlaybackAfterSeek: Bool = false, seekHint: SeekHint? = nil) {
         guard let playingEpisode = currentEpisode() else { return } // nothing to actually seek
 
+        // When using EffectsPlayer we have an issue in which rapidly tapping
+        // skip back results (sometimes) in skipping forward
+        // Here we handle that to avoid this issue
+        // See https://github.com/Automattic/pocket-casts-ios/issues/1950
+        if seekHint == .back {
```

Comment:
Nit: Could we refactor this to a separate method something like `isValidSeek` ?

Reply:
Done in 601a19609556bbf1461e9e4f5913728e2711bc5f